lib/nanoc/base/views/config_view.rb

```diff
@@ -47,5 +47,11 @@ def each(&block)
       @context.dependency_tracker.bounce(unwrap, attributes: true)
       @config.each(&block)
     end
+
+    # @see Hash#dig
+    def dig(*keys)
+      @context.dependency_tracker.bounce(unwrap, attributes: keys)
```

Comment:
`attributes` is a list of top-level attributes that are touched, so this should be `keys.first` instead. Nanoc only tracks dependencies of keys one level deep.

(`keys` will also work, but generate unnecessary dependencies.)